README.md

```diff
@@ -30,12 +32,12 @@ Returns JSON:
 ```json
 [
     {
-        id: 123,
-        name: "McFIT Leer",
+        id: 1674653140,
+        name: "McFIT Hannover-Vahrenheide",
         address: {
-            city: "Leer",
-            zip: "12345",
-            street: "Hauptstrasse 1"
+            city: "Hannover",
+            zip: "30179",
+            street: "Vahrenwalder Str."
```

Comment:
this json had been ill formatted until now, thats we get these ugly red highlights 😄 could you put all the keys in the dictionary in quotes? that should fix it.

```
{
   "id": 123,
   ...
}

Reply:
Ohhhh, yeah, that makes sense. I was already wondering why github is always highlighting it, but that explains it. Thanks! :smile: